CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed 🐞
 
+-   Fix switch button in delta mode [#3023](https://github.com/MaibornWolff/codecharta/pull/3023)
```

Comment:
The button is actually fine. It's just an error that is logged. Users won't notice a difference.